src/parse.rs

```diff
@@ -1162,7 +1197,6 @@ mod tests {
 
     #[test]
     fn unknown_flag() {
-        assert_error("(?<name>[-*_])", "Unknown group flag: (?<");
```

Comment:
Can you add some specific unit tests for named groups in this file too? I'd like to see the error cases covered too (backref before group name defined, etc).

Reply:
Certainly.